src/main/java/com/bobocode/hoverla/bring/context/BeanInitializer.java

```diff
@@ -7,7 +7,13 @@ public class BeanInitializer {
     /**
      * Instantiates beans and injects dependencies in them
      */
-    void initialize(Map<String, BeanDefinition> beanDefinitions) {
-
+    public void initialize(Map<String, BeanDefinition> beanDefinitions) {
+        for (Map.Entry<String, BeanDefinition> entry : beanDefinitions.entrySet()) {
```

Comment:
Looks like a perfect candidate for 'var' instead of 'Map.Entry<String, BeanDefinition> '

Reply:
@DanielChanger yup, makes sense